src/contact.js

```diff
@@ -0,0 +1,52 @@
+
+
+
+const contact = (()=>{
```

Comment:
- [ ] Please remove the extra empty lines from your files. Your code is already tidy but it can get even better!